palm/plugins/dbt/templates/model_docs/model_doc.tpl.md

```diff
@@ -0,0 +1,10 @@
+{{ begin_docs }}
+# {{model_name_humanized}}
+
+**Grain**: {{ grain }}
+
+## Data Product Details
+
+{{ description }}
```

Comment:
does this handle wrapping text? non-blocking

Reply:
No, but right now the description is inut via the CLI, I don't think people will write very long descriptions there. Personally I'd write a short description, then edit the markdown file once it's generated